docs/release_notes/destinations_v2.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+function concatenateRawTableName(namespace, name) {
+  let plainConcat = namespace + name;
+  // Pretend we always have at least one underscore, so that we never generate `_raw_stream_`
+  let longestUnderscoreRun = 1;
+  for (let i = 0; i < plainConcat.length; i++) {
+    // If we've found an underscore, count the number of consecutive underscores
+    let underscoreRun = 0;
+    while (i < plainConcat.length && plainConcat.charAt(i) === '_') {
+        underscoreRun++;
+        i++;
+    }
+    longestUnderscoreRun = Math.max(longestUnderscoreRun, underscoreRun);
+  }
+  return namespace + "_raw" + "_".repeat(longestUnderscoreRun + 1) + "stream_" + name;
+}
+
+// Taken from StandardNameTransformer
+function convertStreamName(str) {
+  return str.normalize('NFKD')
+    .replaceAll(/\p{M}/gu, "")
+    .replaceAll(/\s+/g, "_")
+    .replaceAll(/[^A-Za-z0-9_]/g, "_");
+}
+
+const SqlOutput = ({ destination, defaultMessage }) => {
+  return (
+    <pre id={ "sql_output_block_" + destination }>
```

Comment:
I have no idea what I'm doing :P if you know how to generate an actual code block (with syntax highlighting), let me know!

Reply:
try `<code>`!

Reply:
alas
![image](https://github.com/airbytehq/airbyte/assets/5741425/dfa79fc9-379b-4d0d-ac20-077abe856240)

Reply:
oh, https://docusaurus.io/docs/next/markdown-features/code-blocks#usage-in-jsx looks relevant... but `innerHTML` isn't the right way to update the contents >.>

Reply:
going down a rabbithole of how react works... I think I'm close to figuring this out

Reply:
ayyyy
![image](https://github.com/airbytehq/airbyte/assets/5741425/37b5d632-1304-4c86-83ee-8bee799ee189)

Reply:
Now you are a front end developer too!